transactions_automatic_recurring_transfers.md

```diff
@@ -1,36 +1,51 @@
 # How-To: Automatic Recurring Transfers and Irrevocable Transfers
 
-(Formerly known as Forward Transfers and Standing Orders)
 
-After logging in, select 'Transactions' on the left. You will now see there are three options on this page: Pending, History, and Recurring. Select Recurring. This is where you will see your recurring transfers.
+Per the M-RETS Operating Procedures:
+
+- After a User initiates an Automatic Recurring Transfer (“Transferor”), the Automatic Recurring
+Transfer enters a ‘Pending’ state. The receiving Organization (“Transferee”) then receives an email detailing
+the pending Automatic Recurring Transfer.
+- Each Automatic Recurring Transfer will be set up based on percentage of Certificates or a maximum
```

Comment:
Can we clarify that ART is short for Automatic Recurring Transfer?

Reply:
done